tests/sals/automated_chatflows/test_marketplace_apps.py

```diff
@@ -43,7 +43,7 @@ def test01_wiki(self):
         self.solution_uuid = wiki.solution_id
 
         self.info("Check that the wiki is reachable.")
-        request = j.tools.http.get(f"https://{wiki.domain}", verify=False)
+        request = j.tools.http.get(f"https://{wiki.domain}", verify=False, timeout=5)
```

Comment:
would be better to reference this as a constant? it would be only a single line change for later.